src/vcpkg/commands.edit.cpp

```diff
@@ -66,8 +86,10 @@ namespace
                 ExpandEnvironmentStringsW(widened.c_str(), &result[0], static_cast<unsigned long>(result.size() + 1));
             if (required_size == 0)
             {
-                vcpkg::print2(vcpkg::Color::error, "Error: could not expand the environment string:\n");
-                vcpkg::print2(vcpkg::Color::error, input);
+                msg::print_error(msg::format(msg::msgErrorMessage)
```

Comment:
I think this will print "error: " twice after https://github.com/microsoft/vcpkg-tool/pull/553. That PR also adds additional validation for localization changes, so let's wait until that one gets merged.

Reply:
You're right